README.md

```diff
@@ -300,9 +351,10 @@ A basic server-side application has this form:
 ```python
 import asyncio
 import json
-import server_cp as server
+from micropython_iot import server_cp as server
```

Comment:
this imports from the renamed package "micropython_iot" as "micropython-iot" is not allowed in python.